### PR TITLE
Create 2020-08-26-first-post.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: posts/intjnotinfp/2020-08-26-first-post.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: <intjnotinfp>
```
Comment (eah13) [inline posts/intjnotinfp/2020-08-26-first-post.md]: The `<>` brackets aren't needed. No worries- using brackets in instructions  to indicate a 'variable' should go in there is an idiom you'll become more familiar with over time.

**Review by eah13 — CHANGES_REQUESTED** [review]
I see two problems with the filename of your file: one is the underscore is missing in `_posts` and the second, which was an omission in the instructions, is we need your filename to have something unique in it in addition to your username.

This review says 'request changes' but everygthing's fine. We'll go over all of this in class.

**Comment by ghost** [comment]
@eah13 I changed the file name. Now ready for final review.

**Review by eah13 — APPROVED** [review]
Looks great now!

**Pull request merged**